src/main/java/appeng/tile/misc/TileInscriber.java

```diff
@@ -22,7 +22,6 @@
 import java.io.IOException;
 import java.util.EnumSet;
 import java.util.List;
-
 import javax.annotation.Nullable;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) 'javax.annotation.Nullable' should be separated from previous imports. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)